File: VB/Applications/WpfSerilogVBDI/ViewModels/MainViewModel.vb
Imports LogViewerVB.Core.ViewModels
Imports MvvmVB.Core

Public Class MainViewModel :  Inherits Viewmodel

#Region "Constructor"

    Public Sub New(logViewer As LogViewerControlViewModel)
        Me.LogViewer = logViewer
    End Sub

#End Region

#Region "Properties"

    Public Property LogViewer As LogViewerControlViewModel

#End Region

End Class

File: CSharp/Applications/WpfLoggingAttrDI/ViewModels/MainViewModel.cs
using LogViewer.Core.ViewModels;
using Mvvm.Core;

namespace WpfLoggingAttrDI.ViewModels;

public class MainViewModel : ViewModel
{
    #region Constructor

    public MainViewModel(LogViewerControlViewModel logViewer)
    {
        LogViewer = logViewer;
    }

    #endregion

    #region Properties

    public LogViewerControlViewModel LogViewer { get; }

    #endregion
}